scenarios/01_Highest_Tiers.cfg

```diff
@@ -1519,5 +1519,6 @@
 
     {SUMMONER_LOCK}
     {DEATH_MEHIR}
+    {MEDITATION_EVENT}
```

Comment:
why not [event] in [campaign]

Reply:
At first I thought to do it like this to avoid farming experience in the scenarios of only speaking. But I noticed that they don't have turns, since Mehir's moves recharge until you finish all the tasks, so it's safe to activate this event in these scenarios as well, the condition will never be met.